bok_choy/browser.py

```diff
@@ -201,18 +202,24 @@ def browser_check_func():
             return True, browser_class(*browser_args, **browser_kwargs)
 
         except socket.error as err:
-            LOGGER.debug('Failed to instantiate browser: ' + err.strerror)
+            msg = str(err)
+            LOGGER.debug('Failed to instantiate browser: ' + msg)
+            return False, None
+
+        except WebDriverException as err:
```

Comment:
I'm not sure it's helpful to the user to separate the two as all they would see is the msg anyhow (and "Failed to instantiate browser: " only in the case of a socket error but without seeing the code the user wouldn't know that.)

I think it's preferable to instead combine the two with:
`except (socket.error, WebDriverException) as err:`